pkg/cmd/taskrun/delete_test.go

```diff
@@ -569,6 +687,60 @@ func TestTaskRunDelete(t *testing.T) {
 			wantError:   true,
 			want:        "taskruns.tekton.dev \"nonexistent\" not found",
 		},
+		{
+			name:        "Delete all of task with default --ignore-running",
+			command:     []string{"delete", "-t", "random", "-f", "-n", "ns", "--ignore-running"},
+			dynamic:     seeds[9].dynamicClient,
+			input:       seeds[9].pipelineClient,
+			inputStream: nil,
+			wantError:   false,
+			want:        "All TaskRuns(Completed) associated with Task \"random\" deleted in namespace \"ns\"\n",
+		},
+		{
+			name:        "Delete all of task with explicit --ignore-running true",
+			command:     []string{"delete", "-t", "random", "-f", "-n", "ns", "--ignore-running=true"},
+			dynamic:     seeds[10].dynamicClient,
+			input:       seeds[10].pipelineClient,
+			inputStream: nil,
+			wantError:   false,
+			want:        "All TaskRuns(Completed) associated with Task \"random\" deleted in namespace \"ns\"\n",
```

Comment:
seems to be duplicate as the default value of `--ignore-running` is true.

Reply:
Added it to check the default behavior, if somehow the default behavior changes in future